azure-iot-device/azure/iot/device/provisioning/transport/constant.py

```diff
@@ -0,0 +1,50 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module contains constants related to the transport package.
+"""
+
+__all__ = [
+    "USER_AGENT",
+    "API_VERSION",
+    "SUBSCRIBE_TOPIC_PROVISIONING",
+    "PUBLISH_TOPIC_REGISTRATION",
+]
+
+USER_AGENT = "azure-iot-provisioning-devicesdk" + "/" + "0.0.1"
+"""
+Default interval for polling, to use in case service doesn't provide it to us.
+"""
+DEFAULT_POLLING_INTERVAL = 2000
+
+"""
+api version to use while communicating with service.
+"""
+# API_VERSION = "2018-11-01"
+API_VERSION = "2019-03-31"
+"""
+Default timeout to use when communicating with the service
+"""
+# TODO Reduce 1 zero
+# DEFAULT_TIMEOUT_INTERVAL = 30000
+DEFAULT_TIMEOUT_INTERVAL = 120000
+SUBSCRIBE_TOPIC_PROVISIONING = "$dps/registrations/res/#"
+"""
+The first part of the topic string used for publishing.
+The registration request id (rid) value is appended to this.
+"""
+PUBLISH_TOPIC_REGISTRATION = "$dps/registrations/PUT/iotdps-register/?$rid="
+"""
+The first part of the topic string used for publishing a query request.
+This must be appended by the registration request id (rid) as well as the
+second part which includes an operation id
+"""
+PUBLISH_TOPIC_QUERYING_I = "$dps/registrations/GET/iotdps-get-operationstatus/?$rid="
+"""
+The second part of the topic string used for publishing a query request.
+The operation id value is appended to this.
+"""
+PUBLISH_TOPIC_QUERYING_II = "&operationId="
```

Comment:
Make just a single string that has format fields defined, e.g.

"$dps/registrations/GET/iotdps-get-operationstatus/?$rid={rid}&operationId={operationId}"

Then use the .format method to fill them in.